src/main/java/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import db.Storage;
```

Comment:
Please ensure that the 'db' package is used to hold the Storage class. If it's not already there, move the Storage class to the appropriate package to maintain a clean project structure.